german-iso.lbx

```diff
@@ -11,6 +11,14 @@
                   {betreut von}},
   urlalso      = {{auch verf\"{u}gbar unter}
                   {auch verf\"{u}gbar unter}},
+  director     = {{Direktor}% female: Direktorin
+                  {Direktor}},% FIXME: unsure
+  bydirector   = {{unter der Regie von}%
```

Comment:
"unter der Regie von" sounds a bit stilted to me, especially if we are just talking about a film/TV series. But I know no better alternative.

Reply:
As a general rule I only provide translations in the `.lbx` if I'm sure they are correct. In this case I would probably have commented out the unsure entries completely, so that it is more obvious for language contributors that something is missing: https://github.com/plk/biblatex/commit/3d1955300b2753938c492b828520ccf8a25059b4

If the translation is commented out it is usually clear to a user that the string is missing (there is a warning and the string appears in bold). But if the translation is bad, a user might not catch that.

Reply:
I see, I will replace the google-translated strings with `FIXME: missing`.

BTW, `Regie` would better fit for `bydirector`.